models/action.go

```diff
@@ -746,5 +747,8 @@ func GetFeeds(opts GetFeedsOptions) ([]*Action, error) {
 
 	}
 
+	if opts.Collaborate {
```

Comment:
This will drop other conditions like `opts.IncludeDeleted`.  I think this method should refactor via `builder` otherwise it's still not correct.